PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java

```diff
@@ -193,6 +192,32 @@ private void updatePoints(int position) {
         SessionHistory.totalPoints += answers.get(position).getPoints();
     }
 
```

Comment:
Hey @ValentunSergeev, your work is perfect considering what was required. But you have added dialog box which was not needed. Please remove dialog later when you have time (maybe after gci). I am approving your task for now.

Reply:
Hi @codingblazer I removed this dialog